### ISSUE TITLE
[NPC] Fel Soldier

### ISSUE THREAD

**Issue by gygyti** [TrinityCore/TrinityCore#26764]
### Description

There is to many Fel Soldiers.
![WoWScrnShot_072921_221202](https://user-images.githubusercontent.com/77268574/127552682-46ce341f-4e16-4ca6-b243-dcc5f1dfab53.jpg)

### Expected behaviour

**the server works for 40 days without interruption** 

### Steps to reproduce the problem

Fel Soldier

Entry: 18994

Map: 530, Zone 3483
x: -208; y: 1870; z: 124
(zonex: 71; zoney: 50)

### Branch

3.3.5

### TC rev. hash/commit

fece5bd

### Operating system

 Debian 10 x64,

### Custom changes

none

**Comment by Aokromes** [comment]
> ### Description
> There is to many Fel Soldiers.
> ![WoWScrnShot_072921_221202](https://user-images.githubusercontent.com/77268574/127552682-46ce341f-4e16-4ca6-b243-dcc5f1dfab53.jpg)
> 
> ### Expected behaviour
> **the server works for 40 days without interruption**

not blizzlike :trollface:

**Comment by Jinnaix** [comment]
Permanently summoned by https://tcubuntu.northeurope.cloudapp.azure.com/aowow/?npc=19005#sai-actionlist-1900500

Will stay forever if not killed.

@offl should it be like this? Never saw them myself before.

**Comment by gygyti** [comment]
> 
> 
> Permanently summoned by https://tcubuntu.northeurope.cloudapp.azure.com/aowow/?npc=19005#sai-actionlist-1900500
> 
> Will stay forever if not killed.
> 
> @offl should it be like this? Never saw them myself before.
> 
> /edit:
> 
> > Expected behaviour
> > the server works for 40 days without interruption
> 
> Even blizzard servers don't run for more than 7 days. I don't know why people are so obsessed with uptime.
> My personal opinion:
> Planned restarts are good for your server health and they give space for doing updates and uninterrupted backups.

I agree with you, you should reboot the machine often. In fact, such a long-term operation of the server made it possible to identify, for example, such a problem

**Comment by Jildor** [comment]
Seems smart_script SetActive + pathfinding problem.

**Comment by gygyti** [comment]
I think it has something to do with this 
![изображение](https://user-images.githubusercontent.com/77268574/128245218-728388ee-aaf5-42a2-8db9-72d96eae998a.png)

**Comment by Faq** [comment]
Just wondering how your server is up for 40 days, but in issue description linked revision hash is only 7 days old.

**Comment by gygyti** [comment]
> 
> 
> Just wondering how your server is up for 40 days, but in issue description linked revision hash is only 7 days old. For me it looks you did not provide correct rev. hash.

no no, this is for an example of how long the server can work)
